src/ontology/mondo-ingest.Makefile

```diff
@@ -382,29 +382,35 @@ define build_mondo
 	echo "$$latest_hash" > $(1)
 endef
 
-tmp/mondo_repo_built: .FORCE
+$(TMPDIR)/mondo_repo_built: .FORCE
 	@if [ ! -f $@ ]; then \
 		$(call build_mondo, $@); \
 	else \
 		current_hash=$$(cat $@); \
-		cd tmp/mondo; \
+		cd $(TMPDIR)/mondo; \
 		git fetch origin; \
 		latest_hash=$$(git rev-parse origin/master); \
 		if [ ! "$$current_hash" = "$$latest_hash" ]; then \
 			cd .. && mv mondo mondo-bak && mv mondo_repo_built mondo_repo_built-bak; \
 			cd .. && $(call build_mondo, $@); \
-			rm -rf tmp/mondo-bak tmp/mondo_repo_built-bak; \
+			rm -rf $(TMPDIR)/mondo-bak $(TMPDIR)/mondo_repo_built-bak; \
 		fi; \
 	fi
 
-$(TMPDIR)/mondo.owl: tmp/mondo_repo_built
+$(TMPDIR)/mondo.owl: $(TMPDIR)/mondo_repo_built
 	cp $(TMPDIR)/mondo/src/ontology/mondo.owl $@
 
- $(TMPDIR)/mondo.sssom.tsv: tmp/mondo_repo_built
+# The following target simulates the EMC pipeline. It runs it in exactly the same way
```

Comment:
I think this comment is not supposed to be on `$(TMPDIR)/mondo.sssom.tsv`, but perhaps `$(TMPDIR)/mondo-with-simulated-emc-run.owl`.

Also IDK if `make` `--dry-run` would be helpful at all here. Just throwing that out there since I think we've underutilized it.

Reply:
@joeflack4 thanks for the catch, moved the comment to the right place and improved it a bit.

Not sure about your --dry-run comment in this context! Can you expand what you are suggesting?

Reply:
Yeah I'm not sure either without understanding what your simulation command is exactly doing.

But basically what `--dry-run` does is let you see all of the goals (the dependency chain) that will run as a result of you running `make GOAL` without actually executing them. Maybe that's not helpful here, I'm not sure.